frontend/css/free_home_view.css

```diff
@@ -0,0 +1,73 @@
+.color-div {
+    background-color: white;
```

Comment:
Codacy found an issue: [Expected indentation of 2 spaces (indentation)](https://app.codacy.com/gh/yadevee/yals/pullRequest?prid=6343113)